Does not compile on Windows with latest version of win32

I get the following error when trying to compile 2.4.6 and 2.4.7:
C:\...\fast-logger-2.4.7\System\Log\FastLogger\FileIO.hs:25:26: error:
    Ambiguous occurrence `fILE_APPEND_DATA'
    It could refer to either `System.Win32.File.fILE_APPEND_DATA',
                             imported from `System.Win32.File' at System\Log\FastLogger\FileIO.hs:10:1-24
                          or `System.Log.FastLogger.FileIO.fILE_APPEND_DATA',
                             defined at System\Log\FastLogger\FileIO.hs:18:1

It looks like this was added to Win32 in June:
https://github.com/haskell/win32/commit/84b96632627f758f8af96c340a161b138dae9cde#diff-a99994770dbe04188c1176136aef3d32

Maybe delete the two lines:
https://github.com/kazu-yamamoto/logger/blob/fast-logger-2.4.6/fast-logger/System/Log/FastLogger/FileIO.hs#L17

--- a/fast-logger/System/Log/FastLogger/FileIO.hs
+++ b/fast-logger/System/Log/FastLogger/FileIO.hs
@@ -12,6 +12,11 @@ import Graphics.Win32.Misc (getStdHandle, sTD_OUTPUT_HANDLE, sTD_ERROR_HANDLE)
 import Data.Bits ((.|.))
 
 type FD = HANDLE
+#ifdef !MIN_VERSION_win32(2,3,1,0)
+-- This flag is not defined in System.Win32.File
+fILE_APPEND_DATA :: UINT
+fILE_APPEND_DATA = 0x0004
+#endif
 
 -- This flag is not defined in System.Win32.File
 fILE_APPEND_DATA :: UINT

